compiler/z/codegen/OMRTreeEvaluatorTable.hpp

```diff
@@ -24,780 +24,774 @@
-   TR::TreeEvaluator::idivEvaluator,        // TR::idiv
-   TR::TreeEvaluator::ldivEvaluator,        // TR::ldiv
-   TR::TreeEvaluator::fdivEvaluator,        // TR::fdiv
-   TR::TreeEvaluator::ddivEvaluator,        // TR::ddiv
-   TR::TreeEvaluator::unImpOpEvaluator,        // TR::bdiv
-   TR::TreeEvaluator::unImpOpEvaluator,        // TR::sdiv
-   TR::TreeEvaluator::idivEvaluator,        // TR::iudiv
-   TR::TreeEvaluator::ldivEvaluator,        // TR::ludiv
-   TR::TreeEvaluator::iremEvaluator,        // TR::irem
-   TR::TreeEvaluator::lremEvaluator,        // TR::lrem
-   TR::TreeEvaluator::fremEvaluator,        // TR::frem
-   TR::TreeEvaluator::dremEvaluator,        // TR::drem
-   TR::TreeEvaluator::unImpOpEvaluator,        // TR::brem
-   TR::TreeEvaluator::unImpOpEvaluator,        // TR::srem
-   TR::TreeEvaluator::iremEvaluator,        // TR::iurem
-   TR::TreeEvaluator::inegEvaluator,        // TR::ineg
-   TR::TreeEvaluator::lnegEvaluator,        // TR::lneg
-   TR::TreeEvaluator::fnegEvaluator,        // TR::fneg
-   TR::TreeEvaluator::dnegEvaluator,        // TR::dneg
-   TR::TreeEvaluator::bnegEvaluator,        // TR::bneg
-   TR::TreeEvaluator::snegEvaluator,        // TR::sneg
-   TR::TreeEvaluator::iabsEvaluator,        // TR::iabs
-   TR::TreeEvaluator::labsEvaluator,        // TR::labs
-   TR::TreeEvaluator::iabsEvaluator,        // TR::fabs
-   TR::TreeEvaluator::iabsEvaluator,        // TR::dabs
 
-   TR::TreeEvaluator::ishlEvaluator,        // TR::ishl
-   TR::TreeEvaluator::lshlEvaluator,        // TR::lshl
-   TR::TreeEvaluator::bshlEvaluator,        // TR::bshl
-   TR::TreeEvaluator::sshlEvaluator,        // TR::sshl
-   TR::TreeEvaluator::ishrEvaluator,        // TR::ishr
-   TR::TreeEvaluator::lshrEvaluator,        // TR::lshr
-   TR::TreeEvaluator::bshrEvaluator,        // TR::bshr
-   TR::TreeEvaluator::sshrEvaluator,        // TR::sshr
-   TR::TreeEvaluator::iushrEvaluator,       // TR::iushr
-   TR::TreeEvaluator::lushrEvaluator,       // TR::lushr
-   TR::TreeEvaluator::bushrEvaluator,       // TR::bushr
-   TR::TreeEvaluator::sushrEvaluator,       // TR::sushr
-   TR::TreeEvaluator::integerRolEvaluator,  // TR::irol
-   TR::TreeEvaluator::integerRolEvaluator,  // TR::lrol
-   TR::TreeEvaluator::iandEvaluator,        // TR::iand
-   TR::TreeEvaluator::landEvaluator,        // TR::land
-   TR::TreeEvaluator::bandEvaluator,        // TR::band
-   TR::TreeEvaluator::sandEvaluator,        // TR::sand
-
-   TR::TreeEvaluator::iorEvaluator,         // TR::ior
-   TR::TreeEvaluator::lorEvaluator,         // TR::lor
-   TR::TreeEvaluator::borEvaluator,         // TR::bor
-   TR::TreeEvaluator::sorEvaluator,         // TR::sor
-
-   TR::TreeEvaluator::ixorEvaluator,        // TR::ixor
-   TR::TreeEvaluator::lxorEvaluator,        // TR::lxor
-   TR::TreeEvaluator::bxorEvaluator,        // TR::bxor
-   TR::TreeEvaluator::sxorEvaluator,        // TR::sxor
 
 #define SIGNED true
 #define UNSIGN false
```

Comment:
It seems odd now that these are present. How does the macro expansion work inside of other macros? I'm a bit worried with the layers of macros this now adds complexity. I think we should just inline these values in the 6 or so places that use it and get rid of these `#define` and `#undef` statements.

Reply:
The macro first expands the SIGN and UNSIGN uses, turning it to `true` and `false`, respectively. It's about 13 SIGN and 6 UNSIGN uses. But I agree, inlining the two macros would reduce the complexity.

Reply:
I propose we fold them in this PR to simplify things.